src/pages/events/commands.md

```diff
@@ -105,9 +105,11 @@ The command supports observer events by default. You must perform additional ste
 
 You can also subscribe to a plugin event if it was registered in the `app/etc/config.php` file and subsequently unsubscribed with the [`events:unsubscribe` command](#unsubscribe-from-a-commerce-event). [Register events](./module-development.md#register-events) describes the format of these files.)
 
+You can also create and subscribe to a rule event. Rule events allow you to determine the conditions that the Commerce events client module uses to emit a native or custom events to your application. See [Create rule events](./rule-events.md) for detailed information and examples.
+
 ### Usage
 
-`bin/magento events:subscribe <event_code> --fields=<name1> --fields=<name2>`
+`bin/magento events:subscribe <event_code> --force --fields=<name1> --fields=<name2> --parent <event_code> --rules=<field-name>|<operator>|<value>`
```

Comment:
Maybe it's better not to give `--force` option in this example. As it's not always required to use it. 
I would add two rules to the example to show that it can be multiple options.
```

`bin/magento events:subscribe <event_code> --parent <event_code>  --fields=<name1> --fields=<name2>  --rules=<field-name1>|<operator>|<value2> --rules=<field-name2>|<operator>|<value2>
```

Reply:
This is the usage statement, rather than an example, so I need to show all the options and arguments. I'll add the second instance of `--rules`.